assets/src/setup/pages/choose-reader-theme/index.js

```diff
@@ -96,7 +106,10 @@ export function ChooseReaderTheme() {
 							{ __( 'Unavailable themes', 'amp' ) }
 						</h3>
 						<p>
-							{ __( 'The following themes are compatible but cannot be installed automatically. Please install them manually, or contact your host if you are not able to do so.', 'amp' ) }
+							{ AMP_QUERY_VAR_CUSTOMIZED_LATE
+								? sprintf( __( 'The following themes are not available because your site (probably the active theme) has customized the AMP query var too late (it is set to “%1$s” as opposed to the default of “%2$s”). Please make sure that any customizations done by defining the AMP_QUERY_VAR constant or adding an amp_query_var filter are done before the `plugins_loaded` action with priority 8.', 'amp' ), AMP_QUERY_VAR, DEFAULT_AMP_QUERY_VAR )
```

Comment:
Translator comment missing.

Reply:
Would you prefer surrounding technical jargon such as  with the `code` tag (such as `AMP_QUERY_VAR` and `plugins_loaded`)?

---

The backticks look really weird in my browser:

![image](https://user-images.githubusercontent.com/16200219/86866364-57f2b880-c0c0-11ea-90cc-80882dddaa8a.png)

Reply:
Oops. I meant to remove the backticks because I saw the same weirdness. 

I wasn't sure how best to add `code` in this case. I suppose it requires some `dangerouslySetInnerHTML`?

Reply:
Addressed in 018d8d85798ea96af061ba1a1af98451ab842885